themes/starter.designers/css/style.css

```diff
@@ -4513,16 +4513,9 @@
 .arlo#arlo .font-arial.fs-11 {
   letter-spacing: 0.03em;
 }
-.arlo#arlo .font-montserrat {
-  font-family: 'Montserrat' !important;
-}
 .arlo#arlo .font-georgia {
```

Comment:
Just wondering if we also need to delete font-georgia (as the other fonts were deleted).  Why wasn't it also deleted?

Reply:
The Montserrat font is a Google Font, so that is why it has been removed. Georgia is a system font that doesn't require an external file.